src/pages/admin/AdminHomepage.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
+import AdminPanelSideBar from "../../components/adminPanelSideBar/AdminPanelSideBar";
 
 class AdminHomepage extends React.Component {
     render() {
         return (
-            <div>
-                <h1>Hello Admin!</h1>
+            <div classname="container">
+                <AdminPanelSideBar/>
```

Comment:
This might need to be added to `AdminPanel.js` which is the base component for the entire AdminPanel. AdminHomepage is just a pseudo component right now, which should be shown on `/admin` endpoint.